src/index.html

```diff
@@ -5,8 +5,307 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
```

Comment:
```suggestion
    <link rel="preconnect" href="https://fonts.googleapis.com">
```
if line length is below 80 you can leave attributes on the same line